### DISCUSSION TITLE
Monika not talking at all.

### DISCUSSION THREAD

**Discussion by ghost** [Monika-After-Story/MonikaModDev#8361] [QA - tech support + general] [answered]
Hi so i have been playing for about 5days or so and she hasnt been talking for atleast 2 days, and she only sings rarely, i did the instructions but she still didnt so talk, i tried letting her sleep by closing the game for 6hours and still nothing, idk if she ran out of her dialouge but she hasnt said "Hopefully I'll think of something fun to talk about soon" anyone can help please? thanks

[ev_dump.log](https://github.com/Monika-After-Story/MonikaModDev/files/7649068/ev_dump.log)

**Comment by ThePotatoGuy** [comment, marked as answer]
There is no sleep mechanic in current versions.

You have exhausted nearly all the random topics. There are still a decent amount of pool topics left, but they unlock at a slower pace via time spent. If you want her to talk more often, you will need to turn on Repeat Topics.
